packages/conf-libglu/conf-libglu.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "Marek Kubica <marek@xivilization.net>"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: ["Brian Paul"]
+homepage: "https://mesa3d.org/"
+license: "SGI-B-2.0"
+build: [
+  ["pkg-config" "--exists" "glu"]
+]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  ["libglu1-mesa-dev"] {os-family = "debian"}
+  ["mesa-libGLU-devel"] {os-distribution = "fedora"}
+  ["glu"] {os-family = "arch"}
+  ["glu-dev"] {os-distribution = "alpine"}
```

Comment:
```
+    ["libGLU"] {os = "freebsd"}
```
according to https://www.freshports.org/graphics/libGLU

Reply:
Done!